### DISCUSSION TITLE
Dynamic title  for nested pages?

### DISCUSSION THREAD

**Discussion by yabdab** [statamic/cms#6420] [Q&A]
What is the best way to include parent(s) title in the  ```{{ $title }}```  of the pages?

Currently only the current page title is shown.

For example  /site.com/blog/i-am-the-slug ...

... should have a title of  "Blog - I am the Slug"

**Comment by joshuablum** [comment]
Hey @yabdab,

In your template you should be able to do something like this:
```antlers
<h3>
  {{ parent:title }} - {{ title }}
</h3>
```

Or you could try to edit your Blueprint to parse Antlers for the `title` field. Not 100% this works. Could break stuff.
![CleanShot 2022-08-02 at 6 34 21](https://user-images.githubusercontent.com/3824203/182426988-ea1accad-4e2f-4828-bda6-2566e244f4c2.png)

Then you can use Antlers in the content of your entry. Your entry's title in this case. More on that in our docs [here](https://statamic.dev/antlers#using-antlers-in-content).

Hope that helps :)

**Reply by yabdab** [reply]
I am using Blade, so not sure that will work? 

I tried and $parent is not available in layout.

Also using SEO Pro, which adds a title, but fails to add parent path.

**Reply by joshuablum** [reply]
Thanks for giving it a try, @yabdab!
Wasn't aware that you were using Blade 🔪 instead on Antlers 🦌.

I think you should be able to either use the `$page` variable that get's passed into the view or maybe even the Blade directive for using Statamic Tags.
Using the `$page` variable it would look something like this:
```blade
{{ $page->parent->get('title') }}
```
Just tried it locally and it works 👍

You might also be able to use a Blade directive for this (not 100% sure this works):
```blade
{{ Statamic::tag('parent') }}
```

More on the [`$page` variable in Blade](https://statamic.dev/blade#current-page) and [using using Statamic's Tags in Blade](https://statamic.dev/blade#using-tags-with-blade) in our docs. Having a look a [how to retrieve and manipulate data](https://statamic.dev/extending/data) is also a good idea. Especially when using Blade.
